pages/spec/publish.md

```diff
@@ -30,10 +34,10 @@ A package must specify a valid [SPDX](https://spdx.org/licenses/) license in its
 
 ### Upload rights
 
-A package can only be uploaded by package admins and maintainers. Package admins can grant admin and maintainer rights to other users.
+A package can only be uploaded by package admins and maintainers. Package admins can grant admin and maintainer rights to other users to upload and maintain individual packages. Namespace admins can grant admin and maintainer rights to other users to upload to that namespace and maintain all packages of that namespace.
```

Comment:
A package can only be uploaded by namespace admins and namespace maintainers. 
Namespace admins or maintainers can grant package maintainer rights to other users to upload and maintain individual packages. Namespace admins can grant admin and maintainer rights to other users to upload to that namespace and maintain all packages of that namespace.

Reply:
Ok, so package maintainers cannot upload packages? What can they do then?

Reply:
Package maintainers can maintain individual packages which means they can upload the new versions of the individual packages. 

* Adds users as maintainers to the package.
* Generate upload token for a package.
* Upload new version of the existing package.